route_planner:
  ros__parameters:
    config_yaml_path: "routes/config.yaml"
    csv_base_dir: "routes"
